app/main.py

```diff
@@ -1,5 +1,71 @@
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```

Comment:
You should specify inner type of list. If argument can be None, you should show it in type hint. Fix everywhere
```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] | None = None) -> None:
```